test/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs

```diff
@@ -275,23 +275,35 @@ void SetupWorkloadSetInGlobalJson(out WorkloadSet originalRollback)
         }
 
         [Fact]
-        public void UpdateWorkloadSetViaGlobalJson()
+        public void RestoreWorkloadSetViaGlobalJson()
         {
+            InstallSdk();
```

Comment:
I think you need to remove the call to `InstallSdk` from `SetupWorkloadSetInGlobalJson`, and add it back in `UseGlobalJsonToSpecifyWorkloadSet`.